### PR TITLE
change about section

### PR DESCRIPTION
**Pull request by izmahaerill**
...

### CODE REVIEW

**Review by fiquell — CHANGES_REQUESTED** [review]
Please change all tag!

NOTE:
- Remove all text colors

File: src/components/About.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { Typography } from "@material-tailwind/react";
+import React from "react";
+
+export default function About() {
+  return (
+    <>
+      <div className="pt-12">
+        <div className="flex flex-col gap-10 py-28 px-10 ">
+          <div className="flex flex-col items-center justify-center">
+            <Typography variant="h3" className="pb-5">
```
Comment (fiquell) [inline src/components/About.tsx:10]: Use `mb-3` for consistent code!

File: src/components/About.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+              About <span className="text-primary">Us</span>
+            </Typography>
+            <Typography variant="paragraph" className="lg:w-1/2 text-gray-500 text-center">
+              Maecenas temTypography variant="paragraph"us tellus eget condimentum rhoncus sem quam semper
+              libero pulvinar hendrerit id lorem.
+            </Typography>
+          </div>
+          <div className="container grid lg:grid-cols-3 place-items-center grid-cols-1 gap-5">
```
Comment (fiquell) [inline src/components/About.tsx:18]: Use `md:grid-cols-3` for medium devices

File: src/components/About.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+export default function About() {
+  return (
+    <>
+      <div className="pt-12">
+        <div className="flex flex-col gap-10 py-28 px-10 ">
+          <div className="flex flex-col items-center justify-center">
+            <Typography variant="h3" className="pb-5">
+              About <span className="text-primary">Us</span>
+            </Typography>
+            <Typography variant="paragraph" className="lg:w-1/2 text-gray-500 text-center">
```
Comment (fiquell) [inline src/components/About.tsx:13]: Remove `text-gray-500`

File: src/components/About.tsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+              Maecenas temTypography variant="paragraph"us tellus eget condimentum rhoncus sem quam semper
+              libero pulvinar hendrerit id lorem.
+            </Typography>
+          </div>
+          <div className="container grid lg:grid-cols-3 place-items-center grid-cols-1 gap-5">
+            <div className="flex w-80 flex-col text-3xl">
+              <Typography variant="h2">Lorem ipsum dolor sit amet consectetur</Typography>
+            </div>
+            <div className="flex w-80 flex-col">
+              <Typography variant="paragraph" className="text-xl mb-2">Lorem, ipsum </Typography>
```
Comment (fiquell) [inline src/components/About.tsx:23]: Use `h4`/`h5`/`h6`

File: src/components/About.tsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+              <Typography variant="paragraph" className="text-gray-500">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Atque
+                itaque cupiditate similique ullam porro voluptatum, ex vero?
+                Eaque asperiores, unde earum quia veniam culpa, sint optio
+                quidem aliquam voluptate, sapiente similique provident animi id
+                nemo illum maiores non natus placeat!
+              </Typography>
+            </div>{" "}
+            <div className="flex w-80 flex-col">
+              <Typography variant="paragraph" className="text-xl">Lorem, ipsum</Typography>
```
Comment (fiquell) [inline src/components/About.tsx:33]: Use `h4`/`h5`/`h6`

**Review by fiquell — CHANGES_REQUESTED** [review]
Improve this code!..

File: src/components/About.tsx [hunk 6]
```diff
@@ -0,0 +1,47 @@
+import { Typography } from "@material-tailwind/react";
+import React from "react";
+
+export default function About() {
+  return (
+    <>
+      <div className="pt-12">
```
Comment (fiquell) [inline src/components/About.tsx:7]: Remove `pt-12` and use `mt-32`

File: src/components/About.tsx [hunk 7]
```diff
@@ -0,0 +1,47 @@
+import { Typography } from "@material-tailwind/react";
+import React from "react";
+
+export default function About() {
```
[the remark this answers is not in the document]
Comment (izmahaerill) [reply src/components/About.tsx:27]: it's error when use it

**Pull request merged**